src/symbolic/ellipsoidal_transitions.jl

```diff
@@ -107,51 +115,46 @@ function hasTransition(c, u, Ep::UT.Ellipsoid,subsys::AffineSys,L, S, U, maxRadi
      @objective(model, Min, -ϵ+λ*J)# -tr(C)) #TODO regularization ? 
 
     optimize!(model)
-    #println(solution_summary(model))
+    # println(solution_summary(model))
```

Comment:
Remove print

Reply:
for the moment I'm keeping them, don't worry about the comments in the folder examples, but its size has already decreased a lot ;)

Reply:
All right, but then maybe just in `src/*` we should be careful about comments
Or at least explain why they are still here if they have to